calendarx.d.ts

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+
+export type DateLike = Date | string | number

```

Comment:
Does Calendar also support moment dates? I thought so and they're their own beast (sadly). If that's the case, you'll wanna do something like:
```
import * as React from "react";
import * as moment from "moment";

export type DateLike = Date | string | number | moment.Moment;
```

Reply:
I'm not sure we want to explicitly support moment here. I just want my implementation for next/prev to be equivalent to that of moment, thus the extensive unit tests.

Reply:
Makes sense